azurerm/internal/services/digitaltwins/parse/digital_twins_endpoint.go

```diff
@@ -0,0 +1,55 @@
+package parse
+
+// NOTE: this file is generated via 'go:generate' - manual changes will be overwritten
+
+import (
+	"fmt"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type DigitalTwinsEndpointId struct {
+	SubscriptionId           string
+	ResourceGroup            string
+	DigitalTwinsInstanceName string
```

Comment:
It's autogenerated from Tom's tool, by command: `//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=DigitalTwinsEndpoint -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/group1/providers/Microsoft.DigitalTwins/digitalTwinsInstances/instance1/endpoints/endpoint1`.  Do you think we shall change the name?

Reply:
Since it is auto generated we should prolly leave it alone for now. I will speak to Tom about how to go about maybe exposing a naming override since in some cases the name may not make sense or be unnecessarily long.